client/web-sveltekit/src/lib/search/resultsIndicator/TimeoutMessage.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+    export let isError: boolean
+
+    const CENTER_DOT = '\u00B7' // interpunct
+</script>
+
+<div class={`info-badge ${isError && 'error-text'}`}>
+    <div class="duration-badge">
+        <div class="info-badge duration">Taking too long?</div>
+        <div class="separator">{CENTER_DOT}</div>
+        <div class="action-badge">
+            Use
+            <a href="https://sourcegraph.com/docs/code-search/types/search-jobs" target="_blank"> Search Job </a>
```

Comment:
This should probably link to the docs page of the instance. So maybe just
```suggestion
            <a href="/docs/code-search/types/search-jobs" target="_blank"> Search Job </a>
```

? I'm not sure.

Reply:
It should use `/help`, which redirects to the version of the docs corresponding to the instance

Reply:
Are we sure about the `/help` redirect? It doesn't seem to work.

Reply:
Did you remove the `/docs` portion? This URL redirects for me: `https://sourcegraph.sourcegraph.com/help/code-search/types/search-jobs`